cmd/litcli/main.go

```diff
@@ -1,32 +1,29 @@
 package main
 
 import (
-	"context"
-	"encoding/base64"
 	"fmt"
-	"os"
-	"path/filepath"
-	"strings"
-	"syscall"
-
 	terminal "github.com/lightninglabs/lightning-terminal"
 	"github.com/lightninglabs/lightning-terminal/litrpc"
 	"github.com/lightninglabs/lndclient"
 	"github.com/lightninglabs/protobuf-hex-display/jsonpb"
 	"github.com/lightninglabs/protobuf-hex-display/proto"
 	"github.com/lightningnetwork/lnd"
 	"github.com/lightningnetwork/lnd/lncfg"
+	"github.com/lightningnetwork/lnd/macaroons"
 	"github.com/urfave/cli"
-	"golang.org/x/term"
 	"google.golang.org/grpc"
 	"google.golang.org/grpc/credentials"
-	"google.golang.org/grpc/metadata"
+	"gopkg.in/macaroon.v2"
+	"io/ioutil"
+	"os"
```

Comment:
nit: imports. Sigh, I wish there was a way to tell GoLand to do this split automatically. This happens to me all the time.

Reply:
TIL there is :)